src/gpt-auto-generator/gpt-auto-generator.c

```diff
@@ -225,7 +225,7 @@ static int add_mount(
         assert(where);
         assert(description);
 
-        log_debug("Adding %s: %s %s", where, what, strna(fstype));
+        log_debug("Adding %s: %s fstype=%s", where, what, fstype ?: "(any)");
```

Comment:
time for strany()? ;-)

Reply:
I don't think it's worth the trouble. There'd be just this one user.